pipeline/tasks/protoc_tasks.py

```diff
@@ -325,6 +325,17 @@ def modify_imports(self, go_import_base, line):
 
 
 class GrpcPackmanTask(packman_tasks.PackmanTaskBase):
-    def execute(self, language, api_name, output_dir, packman_flags=[]):
-        arg_list = [language, api_name, '-o', output_dir] + packman_flags
+    def execute(self, language, api_name, output_dir, src_proto_path,
+                import_proto_path, packman_flags=[]):
+        api_name = api_name.replace('-', '/')
+        arg_list = [language, api_name, '-o', output_dir]
+
+        def remove_google_dirs(str):
+            return str.split(api_name)[0]
+        imps = map(remove_google_dirs, import_proto_path)
+        srcs = map(remove_google_dirs, src_proto_path)
+
+        arg_list += [arg for src in srcs for arg in ('-r', src)]
```

Comment:
Do we have an example where there are multiple paths in `src_proto_path`? I wonder if maybe it would be better to loop over them and execute -- it is unexpected just to ignore the input...